Wikibase/Tests/Usage/InArrayUsage.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+// Should use array_key_exists
+in_array( $key, array_keys( $array ) );
+in_array( $key, array_keys );
+
+// Nothing wrong with these
+in_array( $key, $array );
+array_keys( $array );
+array_key_exists( $key, $array );
+in_array( $key ) && array_keys( $array );
+in_array( array_keys( $key ), $array );
```

Comment:
No tests for `array_flip` instead of `array_keys`?

Reply:
Yea, I was a little sloppy. I will add one.

Reply:
Done.